### DISCUSSION TITLE
cart-shipping.php is old?

### DISCUSSION THREAD

**Discussion by choskins-godaddy** [bootscore/bootscore#357] [Q&A] [answered]
The version of BootScore I have installed is 5.2.3.0.

Woocommerce is telling me "Your theme (bootCommerce Child) contains outdated copies of some WooCommerce template files." The file it's point out is /themes/bootscore-main/cart-shipping.php. The version I see from bootcommerce is 3.6.0 and the version from woocommerce is 7.3.0. In doing a file compare, I see that the file has changed quite a bit. 

I can see this as being a big problem. Or, is it?

![image](https://user-images.githubusercontent.com/104801366/212491090-4a85cebf-056c-491e-8bd4-34e28d8d9f30.png)

**Comment by crftwrk** [comment, marked as answer]
Outdated Woo template files are not a big problem, this will happen from time to time again. WooCommerce 7.3 arrived yesterday and this file is already updated and merged https://github.com/bootscore/bootscore/pull/354. Release is next Monday but you can download main branch as well if you do not want to wait.

**Reply by choskins-godaddy** [reply]
Do we just download and zip it and then upload the theme?

**Reply by crftwrk** [reply]
Yes! Press the green button and download zip file.
